catalog/catalog-template.yaml

```diff
@@ -30,6 +33,6 @@ entries:
     schema: olm.bundle
   - image: registry.redhat.io/cluster-observability-operator/cluster-observability-operator-bundle@sha256:0a819f7e634e34e4418e54a2fae0d4f1a4ef77a58bae1350695700272113e282
     schema: olm.bundle
-  - image: quay.io/redhat-user-workloads/cluster-observabilit-tenant/cluster-observability-operator/cluster-observability-operator-bundle@sha256:830a8b81e197a2aa81099dfb252a2edd1c113cac5ba829ef5087bcf930cbe446
+  - image: quay.io/redhat-user-workloads/cluster-observabilit-tenant/cluster-observability-operator/cluster-observability-operator-bundle@sha256:53fb99ba28d89473bf4d884115a175eda7334adbcdc5ee9f635a7ab0fb01961f 
```

Comment:
We need to leave the 1.2.0 entry here (and change it to the prod registry `registry.redhat.io`) and add the new bundle.